packages/code-du-travail-frontend/pages/fiche-service-public.js

```diff
@@ -66,6 +67,9 @@ class Fiche extends React.Component {
           description={description}
           image={ogImage}
         />
+        <Head>
+          <link href={data._source.url} rel="canonical" />
```

Comment:
Since you disallow robots on dev env do we really need the canonical tag ?

Reply:
the canonical stuff is used to instruct search engine that the content here come from another site.

Reply:
So you are telling google that it should index pages on the other domains and not ours ? That's very opinionated in my opinion ! Is that really what we want ?

Reply:
> So you are telling google that it should index pages on the other domains and not ours ? That's very opinionated in my opinion ! Is that really what we want ?

I don't intend to says that. Just that the content is coming from another site